backend-services/src/data_service/schema/entities.py

```diff
@@ -35,3 +35,17 @@ class Invite(Model):
 
     code: str
     redeemed: bool = False
+
+
+class OtpLimit(Model):
```

Comment:
I think there should be two models because there are two different triggers for OTP:
1. If the amount sent is greater than the threshold limit.
2. If a wallet address has been saved for 2FA, regardless of the limit, 2FA should be triggered.